.github/workflows/automatically-update-test-links.yml

```
@@ -25,7 +25,14 @@ jobs:
             title: "Image-Bot: New images available"
             body: |
                 New images are available for update
-            commit-message: Image-Bot - Image updates
+            commit-message: |
+                Image-Bot - Image updates
+
+                Changelog: Title
+            committer: mender-test-bot <mender-test-bot@mender.io>
+            author: mender-test-bot <mender-test-bot@mender.io>
```

This e-mail address does not exist, maybe use
```suggestion
            committer: mender-test-bot <mender@northern.tech>
            author: mender-test-bot <mender@northern.tech>
```

---

Are you really sure this should have a `Changelog`? :thinking: 
It just seems a bit out of place to me

---

While I don't have an strong opinion, I don't think it deserves a changelog, no. There is absolutely nothing changing product code, only testing images. I didn't comment as I thought you had a reason for it and I wanted to respect that :)

---

I just thought I remembered it was asked for previously, but just a brain-fart appareantly. 

Thx

---

Mmm. I did ask about a Changelog tag, in order to pass the commit checks, but I meant "Changelog: None" kind of tag. Maybe it was my bad there, then.

---

And now, the :heavy_check_mark: ?